src/gobapi/storage.py

```diff
@@ -277,8 +278,10 @@ def get_entities(catalog, collection, offset, limit, view=None):
     all_entities = session.query(table)
 
     if view is None:
-        # The default result is without deleted items
-        all_entities = all_entities.filter_by(_date_deleted=None)
+        # The default result is where expiration date is in the future or empty
+        all_entities = all_entities.filter(or_(
+            getattr(table, FIELD.EXPIRATION_DATE) > datetime.datetime.now(),
```

Comment:
I propose to have this code in a method to prevent duplication of logic about not being expired (or otherwise stated, being active)